packages/ui-components/src/Icon/Icon.module.scss

```diff
@@ -1,68 +1,61 @@
 @import "../styles/tokens.scss";
-
 .icon {
-  display: inline-flex;
```

Comment:
This was removed because this assumes icon will be rendered with another element for it to be inline with something else -> sine we want the Icon component to resemble an image, we should be passing this attribute to whatever renders the Icon component.

Has been tested.

Reply:
👍🏼